### PR TITLE
- Fixup unit tests

### PR DESCRIPTION
**Pull request by johnsonw**
- Integrate jest
- Remove proxyquire
- Leave existing integration tests but make sure they are disabled. Now
that we are using postgres we can no longer use integration tests.
- Fix a couple of copyright locations from Intel -> DDN
- Update package-lock.json
- add "query" function to db-utils. All queries should go through
db-utils

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: db-utils.js [hunk 1]
```diff
@@ -20,14 +20,21 @@ const stream = highland((push, _) => {
...
+    return c.query("LISTEN table_update").catch(e => console.error("got error", e));
   });
 });
 
 exports.viewer = broadcaster(stream);
+
+const query = (query, transform) =>
+  pool
+    .query(query)
+    .then(r => r.rows[0])
```
Comment (jgrund) [inline db-utils.js]: Why are we getting the first row?

File: db-utils.js [hunk 2]
```diff
@@ -20,14 +20,21 @@ const stream = highland((push, _) => {
...
+    .then(x => transform(x));
```
Comment (jgrund) [inline db-utils.js]: Why do we need to embed the transform in this fn?

File: socket-router/routes/host-route.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    stream
+      .tap(x => console.error(x))
+      .errors(pushSerializeError)
+      .through(through.unchangedFilter)
+      .each(x => resp.socket.emit(req.messageName, x));
+
+    next(req, resp, stream);
+  });
+};
```
Comment (jgrund) [inline socket-router/routes/host-route.js]: This shouldn't be in this PR.

File: db-utils.js (hunk 1 above)
Comment (johnsonw) [reply db-utils.js]: I think what I could do is something like this:
```
pool
  .query(query)
  .then(r => r.rows)
  .then(fp.map(transform));
```

File: db-utils.js (hunk 2 above)
Comment (johnsonw) [reply db-utils.js]: I could go either way on this. I liked the idea of passing the transform function and getting back the transformed data. If you don't like it though i'm not opposed to changing it.

File: socket-router/routes/host-route.js (hunk 3 above)
Comment (johnsonw) [reply socket-router/routes/host-route.js]: I was certain I removed this file... I'll remove it.

File: db-utils.js (hunk 2 above)
Comment (jgrund) [reply db-utils.js]: I'm not sure there's any advantage since you can just chain off the return.

File: db-utils.js (hunk 1 above)
Comment (jgrund) [reply db-utils.js]: meh, `pool.query(query)` is enough I think.

Comment (johnsonw) [reply db-utils.js]: fair enough. I'll update.

File: db-utils.js (hunk 2 above)
Comment (johnsonw) [reply db-utils.js]: i'll update

**Pull request merged**